types/node-downloader-helper/index.d.ts

```diff
@@ -39,7 +62,11 @@ export enum DH_STATES {
 }
 
 export interface DownloaderHelper {
-    on(event: 'start' | 'download' | 'end' | 'pause' | 'resume' | 'stop' | string, callback: () => void): this;
+    on(event: 'start' | 'timeout' | 'pause' | 'stop' | string, callback: () => void): this;
+    on(event: 'download' | 'end', callback: (downloadInfo: DownloadInfo) => void): this;
```

Comment:
According to the module's README, it looks like the `DownloadInfo` objects from the **download** and the **end** events are slightly different.

The `DownloadInfo` objet for a **download** event has a `isResumed` property that doesn't exist for the **end** event, while the **end** event has a `onDiskSize` property that doesn't exist in the **download** event.

These differences could induce uncaught errors during development if not properly added to the typescript definitions file.